src/main/java/net/mcreator/element/converter/v2023_4/WaterEntitiesConverter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.converter.v2023_4;
+
+import com.google.gson.JsonElement;
+import net.mcreator.element.GeneratableElement;
+import net.mcreator.element.converter.IConverter;
+import net.mcreator.element.parts.procedure.LogicProcedure;
+import net.mcreator.element.types.LivingEntity;
+import net.mcreator.workspace.Workspace;
+
+public class WaterEntitiesConverter implements IConverter {
+	@Override
+	public GeneratableElement convert(Workspace workspace, GeneratableElement input, JsonElement jsonElementInput) {
+		LivingEntity entity = (LivingEntity) input;
+
+		if (entity.waterMob)
+			entity.breatheUnderwater = new LogicProcedure(null, true);

```

Comment:
Given I just saw similar case in another PR where procedure fields could get null values, do we check if these values are null? If not, I'd change the main part here to:
```
		entity.breatheUnderwater = new LogicProcedure(null, entity.waterMob);
		entity.pushedByFluids = new LogicProcedure(null, true);
```